docusaurus/docs/iOS/03-guides/16-noise-cancellation.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Noise Cancellation
+description: Documentation on integrating noise cancellation.
+---
+
+Noise Cancellation capabilities of our [iOS Video SDK](https://getstream.io/video/sdk/ios) can be enabled by installing our NoiseCancellation package. Under the hood, this package uses the technology developed by [krisp.ai](https://krisp.ai).
+
+## Installation
+
+### Add the SDK to your project
+
+To add StreamVideo SDK, open Xcode and follow these steps:
+
+- In Xcode, go to File -> "Add Packages..."
+- Paste the URL https://github.com/GetStream/stream-video-noise-cancellation-swift
+- In the option "Dependency Rule" choose "Up to the Next Major Version," in the single text input next to it. The field will be populated automatically with the latest version.
+
+- Choose "Add Package" and wait for the dialog to complete.
+- Select `StreamVideoNoiseCancellation` and add it in your project.
+
+### Integration
+
+Our iOS SDK provides a utility component that make the integration smoother.
+
+- `NoiseCancellationProcessor`: the object that comes from the `StreamVideoNoiseCancellation` and performs the noise cancellation operation.
+- `NoiseCancellationFilter`: an object that conforms to `StreamVideo.AudioFilter` and perform all tasks requires for the noise cancellation session (e.g request start/stop).
+
+
+```swift
+// Firstly you initalize the processor.
+let processor = NoiseCancellationProcessor()
+
+// Secondly you instantiate the NoiseCancellationFilter. You can use any name, but it needs to be uniqueq compared to other AudioFilters you may be using.
+let noiseCancellationFilter = NoiseCancellationFilter(
+    name: "noise-cancellation",
+    initialize: processor.initialize,
+    process: processor.process,
+    release: processor.release
+)
+```
+
+Once you are able to create a `NoiseCancellationFilter` you can rely on `Call`'s API and state to toggle the filter status and also observe the features availability.
+
+#### Feature availability
+
+`Call.state.settings` contains the `noiseCancellation` configuration. The configuration contains a mode property that you look into, to determine the feature's availability:
+
+- `.available`
+The featue has been enabled on the dashboard and it's available for the call. In this case, you are free to present any noise cancellation toggle UI in your application.
```

Comment:
Is this raw from the dashboard or do we check also if the package is integrated? If the former is the case, maybe add a note: provided you have integrated the package?

Reply:
So, this is based only on dashboard. The reason i'm not mentioning here the integration is because, the way we implemented we actually hiding completely the need for StreamVideoNoiseCancellation. Everything is abstracted on the NoiseCancellationFilter you pass on VideoConfig. IMO it makes sense to mention the filter on VideoConfig but may not `StreamVideoNoiseCancellation` as people can choose their own integrations for it.